src/components/stickyFooter/tests/StickyFooter.spec.tsx

```diff
@@ -1,16 +1,34 @@
 import {shallow} from 'enzyme';
 import * as React from 'react';
 import {StickyFooter} from '../StickyFooter';
+import * as styles from '../StickyFooter.scss';
 
 describe('StickyFooter', () => {
     it('should render without error', () => {
-        expect(() => shallow(<StickyFooter />)).not.toThrow();
-        expect(() => shallow(<StickyFooter className='someclass' />)).not.toThrow();
+        expect(() => shallow(<StickyFooter isOpened={true} />)).not.toThrow();
+        expect(() => shallow(<StickyFooter className='someclass' isOpened={true} />)).not.toThrow();
+        expect(() => shallow(<StickyFooter id='oyeah' isOpened={true} />)).not.toThrow();
     });

```

Comment:
You should add tests for isOpened={false} or simply `expect(() => shallow(<StickyFooter />)).not.toThrow();`